setup.py

```diff
@@ -7,27 +7,32 @@
 
 chdir(here)
 
+
 def readme():
     try:
         with open(path.join(here, 'README.md'), encoding='utf-8') as f:
             return f.read()
     except FileNotFoundError:
         return ""
 
+
 setup(
     name='chromewhip',
 
-    version='0.3.4',
+    version='0.3.5',
 
     description='asyncio driver + HTTP server for Chrome devtools protocol',
     long_description=readme(),
+    # This project was originally forked from https://github.com/chuckus/chromewhip, but the version included here
+    # has been developed subsequently by Cosive, while the original version doesn't appear to have been maintained in
+    # some time.
     # The project's main homepage.
-    url='https://github.com/chuckus/chromewhip',
-    download_url='https://github.com/chuckus/chromewhip/archive/v0.3.4.tar.gz',
+    url='https://github.com/Cosive/chromewhip',
+    download_url='https://github.com/Cosive/chromewhip/archive/refs/tags/chrome-96.0.4664.110.tar.gz',
```

Comment:
This download_url will be brittle, and we're not publishing this module to PyPI, so there's no value added here.

I strongly recommend leaving this (and all files sourced from a third-party repo) as-is - unmodified from the original source - unless a change is strictly required to minimise merge fights. Alternatively a (more) stable download URL could be set.

```suggestion
    download_url='https://codeload.github.com/Cosive/chromewhip/tar.gz/refs/heads/master',
```

Reply:
I'm moderately certain that the old download URL listed here was the one that was actually being used when downloading Chromewhip for use in other software (I can't remember precisely why I think that now, though).  Thus, it was quite important to update the URL from what it had been, i.e. an old one on the repo we don't control.

Your proposed new URL does seem likely to be the better choice, though.